OpenNefia.Content/Resources/Locale/en_US/Elona/Gender.lua

```diff
@@ -0,0 +1,12 @@
+﻿Elona.Gender = {
+	Sex = {
```

Comment:
Maybe `Names` instead (I would like to standardize on `gender` instead of `sex` everywhere).

Reply:
changed in 76f6762